### PR TITLE
Minor updates to Greek language.

### PR DESCRIPTION
**Pull request by lourdas**

### CODE REVIEW

File: framework/messages/el/yii.php [hunk 1]
```diff
@@ -114,4 +83,32 @@
...
+    '{nFormatted} KB' => '{nFormatted} KB',
+    '{nFormatted} KiB' => '{nFormatted} KiB',
+    '{nFormatted} MB' => '{nFormatted} MB',
+    '{nFormatted} MiB' => '{nFormatted} MiB',
+    '{nFormatted} PB' => '{nFormatted} PB',
+    '{nFormatted} PiB' => '{nFormatted} PiB',
+    '{nFormatted} TB' => '{nFormatted} TB',
+    '{nFormatted} TiB' => '{nFormatted} TiB',
+    '{nFormatted} {n, plural, =1{byte} other{bytes}}' => '{nFormatted} {n, plural, =1{byte} other{bytes}}',
+    '{nFormatted} {n, plural, =1{gibibyte} other{gibibytes}}' => '{nFormatted} {n, plural, =1{gibibyte} other{gibibytes}}',
```
Comment (samdark) [inline framework/messages/el/yii.php]: Should be translated as well, right?

Comment (lourdas) [inline framework/messages/el/yii.php]: Which words from the quoted lines should be translated? If you're referring to the measurement units (bytes, kilobytes, etc.) I leave them as is, they do not have a corresponding Greek term.

Comment (samdark) [inline framework/messages/el/yii.php]: I wasn't aware of it so just asked.

**Comment by samdark** [comment]
Thanks!

**Pull request merged**